The source code highlighter "source-highlight" is not automatically
available on Solaris, so substitute something that is.

This change will not be passed back upstream.

--- asciidoc-9.0.2/asciidoc.conf
+++ asciidoc-9.0.2/asciidoc.conf
@@ -22,7 +22,7 @@
 # Uncomment to use xhtml11 quirks mode CSS.
 #quirks=
 # HTML source code highlighter (source-highlight, pygments or highlight).
-source-highlighter=source-highlight
+source-highlighter=pygments
 # Uncomment to use deprecated quote attributes.
 #deprecated-quotes=
 empty=
